priv/repo/migrations/20170419210307_create_user.exs

```diff
@@ -0,0 +1,15 @@
+defmodule RemoteRetro.Repo.Migrations.CreateUser do
+  use Ecto.Migration
+
+  def change do
+    create table(:users) do
+      add :google_user_info, :jsonb
```

Comment:
@vanderhoop I personally feel like jsonb makes sense because the data returned from Google shouldn't be editable and it also might change in the future, especially if we add more scopes.  Storing the data unstructured makes it simple to store any extra data in the future without modifying the schema.  The only reason we made email it's own field is because we needed a unique index.